decidim-core/app/controllers/decidim/devise/passwords_controller.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
```

Comment:
I mean, we don't need this as we don't have any string literal here, right? (nor in any of the other controllers that have this comment)